rosdep/python.yaml

```
@@ -7327,6 +7327,20 @@ python3-openhsi-pip:
   ubuntu:
     pip:
       packages: [openhsi]
+python3-overrides:
```

Since this is a pip-only key, this should have a `-pip` suffix:

```suggestion
python3-overrides-pip:
```